packages/wallets/webpack.config.js

```diff
@@ -168,6 +168,12 @@ module.exports = function (env) {
                         priority: -20,
                         reuseExistingChunk: true,
                     },
+                    components: {
+                        test: module => {
+                            return module.resource && module.resource.includes('src/components/Base');
+                        },
+                        name: 'components',
+                    },
```

Comment:
Added `components/Base` as a cache group to allow other chunks to reuse the base components instead of including it inside their chunk